File: Sources/01XXX/1357/1357.pas
(*
 *  BOJ 1357번 Pascal 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 340 KB / 131,072 KB
 *  소요 시간 : 0 ms / 2,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 *)

VAR X,Y:LONGINT;
FUNCTION REV(N:LONGINT):LONGINT;
VAR S,R:LONGINT;
BEGIN
	S:=0;
	WHILE (N>0) DO
	BEGIN
		R:=N MOD 10;
		N:=N DIV 10;
		S:=S*10+R;
	END;
	REV:=S;
END;
BEGIN
	READ(X,Y);
	WRITELN(REV(REV(X)+REV(Y)));
END.

File: Sources/01XXX/1357/1357.java
/**
 *  BOJ 1357번 Java 소스 코드
 *  작성자 : 동동매니저 (DDManager)
 *
 *  ※ 실행 결과
 *  사용 메모리 : 14,308 KB / 278,528 KB
 *  소요 시간 : 100 ms / 5,000 ms
 *
 *  Copyright 2019. DDManager all rights reserved.
 */

import java.util.*;
public class Main{
	public static void main(String args[]){
		Scanner sc=new Scanner(System.in);
		int x=sc.nextInt();
		int y=sc.nextInt();
		System.out.println(rev(rev(x)+rev(y)));
		sc.close();
	}
	private static int rev(int n){
		return Integer.parseInt(new StringBuilder(""+n).reverse().toString());
	}
}